uber/site_sections/accounts.py

```diff
@@ -21,7 +21,7 @@ def index(self, session, message=''):
             'all_attendees': sorted([
                 (id, '{} - {}{}'.format(name.title(), c.BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
                 for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
-                                    .filter(Attendee.first_name != '').all()
+                                    .filter(Attendee.first_name != '').filter(Attendee.badge_status not in [c.INVALID_STATUS, c.DEFERRED_STATUS]).all()
```

Comment:
Perhaps add this explanation as an inline comment in the code?

> On Oct 15, 2015, at 9:27 PM, Victoria Earl <notifications@github.com> wrote:
> 
> In uber/site_sections/accounts.py:
> 
> > @@ -21,7 +21,7 @@ def index(self, session, message=''):
> >              'all_attendees': sorted([
> >                  (id, '{} - {}{}'.format(name.title(), c.BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
> >                  for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
> > -                                    .filter(Attendee.first_name != '').all()
> > +                                    .filter(Attendee.first_name != '').filter(Attendee.badge_status not in [c.INVALID_STATUS, c.DEFERRED_STATUS]).all()
> This is one place where I didn't use valid_attendees() because:
> 
> We also want to exclude Deferred attendees
> This selects only specific columns in the database
> —
> Reply to this email directly or view it on GitHub.
>